sherpa-ncnn/c-api/c-api.h

```diff
@@ -113,10 +113,19 @@ typedef struct SherpaNcnnRecognizerConfig {
 } SherpaNcnnRecognizerConfig;
 
 typedef struct SherpaNcnnResult {
+  // Recognized text
   const char *text;
+
+  // Pointer to continuous memory which holds string based tokens
+  // which are seperated by \0
   const char *tokens;
+
+  // Pointer to continuous memory which holds timestamps which
+  // are seperated by \0
   float* timestamps;
```

Comment:
Please fix the doc about `timestamps`. It is not a string and does not contain `\0`.